frontend/pages/hosts/ManageHostsPage/components/RunScriptBatchModal/RunScriptBatchModal.tsx

```diff
@@ -91,8 +161,12 @@ const RunScriptBatchModal = ({
             runByFilters
               ? totalFilteredHostsCount.toLocaleString()
               : selectedHostIds.length.toLocaleString()
-          } hosts, or will run as each host comes online. See host details for individual results.`
+          } host${selectedHostIds.length === 1 ? "" : "s"}, or will run ${
```

Comment:
Looks like we want to update this message

Reply:
Instead with a link to the batch page, instead of redirecting

Reply:
ah i see you're right, will update